firewatch-base-configs/opp/lp-interop-vsphere.json

```
@@ -0,0 +1,15 @@
+{
+  "failure_rules":
+    [
+      {"step": "ipi-vsphere-pre", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}},
+      {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "group": {"name": "opp-tests", "priority": 1}},
+      {"step": "acm-mch", "failure_type": "all", "classification": "MCH install", "group": {"name": "opp-tests", "priority": 2}},
+      {"step": "acm-policies-openshift-plus-setup", "failure_type": "all", "classification": "Policies setup failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 3}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-policies-openshift-plus", "failure_type": "all", "classification": "Policies testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 3}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-tests-obs", "failure_type": "all", "classification": "OBS testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "cucushift-installer-check-cluster-health", "failure_type": "all", "classification": "Cucushift testcase failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5}, "jira_epic": "!default", "jira_assignee": "!default"},
+      {"step": "acm-must-gather", "failure_type": "pod_failure", "classification": "ACM must-gather failure", "group": {"name": "opp-tests", "priority": 2}},
+      {"step": "mce-must-gather", "failure_type": "pod_failure", "classification": "MCE must-gather failure", "group": {"name": "opp-tests", "priority": 2}},
+      {"step": "openshift-extended-test", "failure_type": "pod_failure", "classification": "OCP Observability failure", "jira_project": "!default", "jira_component": ["!default"], "group": {"name": "opp-tests", "priority": 5, "jira_epic": "!default", "jira_assignee": "!default"}}
+    ]
```

```suggestion
    ],
  "success_rules":
    [
      {"jira_project": "CSPIT", "jira_additional_labels": ["!default"]}
    ]
```
Should this also have success rules?